lib/components/gene/__tests__/__snapshots__/about-test.js.snap

```diff
@@ -143,15 +143,15 @@ exports[`test shows trending artists correctly 1`] = `
             Object {
               "margin": 5,
               "paddingBottom": 20,
-              "width": 157,
+              "width": 1,
             }
           }>
           <ImageView
             imageURL=""
             style={
               Object {
-                "height": 105,
```

Comment:
This was happening because the layout would rely on `Dimensions`, which would assume iPhone dimensions if `window` returned nothing.